serverless/installing_serverless/installing-openshift-serverless.adoc

```diff
@@ -45,8 +45,8 @@ The following limitations apply to all {ServerlessProductName} deployments:
 
 For more advanced use-cases such as logging or metering on {product-title}, you must deploy more resources. Recommended requirements for such use-cases are 24 CPUs and 96GB of memory.
 
-// If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200m - 2GB of memory for each replica of the Knative Serving control plane.
-// HA is enabled for some Knative Serving components by default. You can disable HA by following the documentation on xref:../../serverless-config-HA.adoc#serverless-config-HA[Configuring High Availability components].
+If you have high availability (HA) enabled on your cluster, this requires between 0.5 - 1.5 cores and between 200m - 2GB of memory for each replica of the Knative Serving control plane.
```

Comment:
Double checking that this is correct - should it be "200MB - 2GB" of memory?

Reply:
Should be right since this has been through a bunch of reviews, but checking with eng now.